application/models/Survey.php

```diff
@@ -1165,10 +1165,11 @@ public function getRunning()
             $bExpired = ($oStop < $oNow);
             $bWillRun = ($oStart > $oNow);
 
-            $sStop = convertToGlobalSettingFormat($sStop);
+            $sStop = $sStop!=null ? convertToGlobalSettingFormat($sStop) : null;
             $sStart = convertToGlobalSettingFormat($sStart);
 
             // Icon generaton (for CGridView)
+            $sIconRunNoEx = '<a href="' . App()->createUrl('/surveyAdministration/view/surveyid/' . $this->sid) . '" class="survey-state" data-toggle="tooltip" title="' . sprintf(gT('End: Never')) . '"><span class="fa  fa-play text-success"></span><span class="sr-only">SS' . sprintf(gT('End: Never')) . '</span></a>';
```

Comment:
No need to sprintf

Reply:
Sorry, now I understn this. I overlooked it.